File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/trig/TanZ.java
package net.fwitz.math.main.binary.complex.functions.trig;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class TanZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("tan(z)")
                .computeFn(Complex::tan)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/trig/TanZ.kt
package net.fwitz.math.main.binary.complex.functions.trig

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object TanZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("tan(z)")
        .computeFn(Complex::tan)
        .render()
}